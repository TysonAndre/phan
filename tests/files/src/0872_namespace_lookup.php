<?php
namespace Foo\A;
use Foo\B;
function test(): B\B {
    return null;
}
