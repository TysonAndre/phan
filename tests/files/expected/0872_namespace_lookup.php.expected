%s:4 PhanUndeclaredTypeReturnType Return type of test() is undeclared type \Foo\B\B (Did you mean bool)
%s:5 PhanTypeMismatchReturnReal Returning null of type null but test() is declared to return \Foo\B\B
